### DISCUSSION TITLE
Menu items can be selected briefly using tablet after cmd-tabbing or 3-finger swiping on MacOS

### DISCUSSION THREAD

**Discussion by Non-Binary-Programmer** [ppy/osu#20121] [Q&A]
### Type

Game behaviour

### Bug description

Reproduction steps (3-finger swipe):
1. Hover over a menu item, preferably a large one (such as the area outside the settings menu that closes the menu)
2. 3 finger swipe away
3. Click any time before the new window enters focus without moving the cursor

Reproduction steps (cmd-tab):
1. Hover over a menu item, preferably a large one (such as the area outside the settings menu that closes the menu)
2. Cmd-tab away
3. Click.

The cmd-tab method is somewhat inconsistent, and might take a few tries. You can also input multiple clicks after cmd-tabbing or 3-finger swiping once

### Version

2022.902.1

### Logs

[database.log](https://github.com/ppy/osu/files/9483496/database.log)
[network.log](https://github.com/ppy/osu/files/9483497/network.log)
[performance.log](https://github.com/ppy/osu/files/9483498/performance.log)
[runtime.log](https://github.com/ppy/osu/files/9483499/runtime.log)

**Comment by peppy** [comment]
Please provide a video showing what you're seeing.

**Reply by Non-Binary-Programmer** [reply]
Did you see my message below? I'm sorry if you did and this is unnecessary; I'm not very familiar with Github discussions.

**Reply by smoogipoo** [reply]
Yes, we all get notifications for this. It's low priority at the moment.

**Comment by Non-Binary-Programmer** [comment]
https://user-images.githubusercontent.com/100527514/188295638-9102c754-9c20-44b2-ad88-4c22e1299f2b.mov

I'm in the settings menu and cmd-tab to my chrome window. I then click, and hear the sound of the settings menu closing. Then, I cmd-tab back to osu, and the settings menu is closed.
